### DISCUSSION TITLE
Search not working when the search field is relational.

### DISCUSSION THREAD

**Discussion by alan-cleveston** [directus/directus#11751] [Archive]
When trying to use the search in a field that is relational, the search does not work, it only works if the search field is an input

### To Reproduce

To reproduce the error just use a relational field in a content and try to search for something in the relational field

https://user-images.githubusercontent.com/82655156/154826295-fd9ce390-dfa7-494a-bd65-2d8718bab313.mp4

### What version of Directus are you using?

9.5.2

### What version of Node.js are you using?

16.14.0

### What database are you using?

Postgres 13.4

### What browser are you using?

Chrome

### What operating system are you using?

Linux

### How are you deploying Directus?

Docker

**Comment by rijkvanzanten** [comment]
This is correct. Search uses a plain text search (using `LIKE` in sql) on the parent items. Use the filtering options to search through nested values 👍🏻

**Reply by alan-cleveston** [reply]
Understood, thank you 👍🏻

**Comment by hermesalvesbr** [comment]
This is not practical for a newbie user. This is possible to make a new search field by extensions?

**Reply by rijkvanzanten** [reply]
Not as of right now. That being said, we are looking into the possibility of making the "search engine" an extension type, so you could integrate with a custom search engine or a third party one like Algolia

**Reply by hermesalvesbr** [reply]
If the data is visible on the screen, the user uses the search box only to filter what he wants to see.

At the moment it is not a super search that would help, it would be a simple filter of what is being exposed in the collection table.

**Reply by victorhsb** [reply]
This is exactly what i was searching for. are you guys still looking into it? custom search engines would solve it. i'm having a similar problem with relational searches AND the removal of accented characters on search.
